_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: "Iterating to achieve my first posit::conf talk"
```

Comment:
I feel the word "acceptance" would help understand what it means.

Reply:
added, thank you!